v2/array/array.go

```diff
@@ -41,10 +43,13 @@ func IntersectString(arrs ...[]string) (result []string) {
 	return
 }
 
+const oneOccurence = 1
+
 func DifferenceString(arrs ...[]string) (result []string) {
 	result = []string{}
+
 	for key, count := range countMap(arrs...) {
-		if count == 1 {
+		if count == oneOccurence {
```

Comment:
Are you sure that this is needed? it is not complaining on the one i v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go.

In this case I would say that a nolint would be clearer.

Reply:
the linter failed for this

how can a nolint be clearer?

Reply:
I get suspicious when I read the oneOccurence thing and feel the urge to check that it is really a 1.